connectivity/check/policy.go

```diff
@@ -248,6 +252,13 @@ var (
 		DropReasonFunc: defaultDropReason,
 	}
 
+	// ResultDropAuthRequired expects a dropped flow with auth required as reason.
+	ResultDropAuthRequired = Result{
+		Drop: true,
+		// IngressDrop:    true,
```

Comment:
Do we expect to ever use this? I'd either add a TOD comment with an explanation on why it is commented out or drop it in case we don't expect to use it.

Reply:
this was a leftover of my experiments let me fix that